.azure-pipelines/templates/changes-include.yml

```diff
@@ -0,0 +1,21 @@
+parameters:
+  paths: ''
+
+steps:
+- bash: |
+    echo '##vso[task.setvariable variable=hasChanges]true'
```

Comment:
What is this magic?

Reply:
Definitely put a comment on it since most of us are not going to have any idea what this does.

Reply:
That's how azure pipelines passes variables between steps.